task/reindex.go

```diff
@@ -317,6 +317,20 @@ func transformZebedeeDoc(ctx context.Context, tracker *Tracker, errChan chan err
 	}
 }
 
+func retrieveTopicsMap(ctx context.Context, enabled bool, serviceAuthToken string, topicClient topicCli.Clienter) chan map[string]Topic {
+	topicsMapChan := make(chan map[string]Topic, 1)
+
+	go func() {
+		defer close(topicsMapChan)
+		if enabled {
+			topicsMap := LoadTopicsMap(ctx, serviceAuthToken, topicClient)
+			topicsMapChan <- topicsMap
+		}
```

Comment:
Add some logging to show progress and help us to debug if anything goes wrong.
```suggestion
		if enabled {
			topicsMap := LoadTopicsMap(ctx, serviceAuthToken, topicClient)
			topicsMapChan <- topicsMap
			log.Info(ctx, "finished retrieving topics map", log.Data{"map_size": len(topicsMap)})
		}
		else {
		     log.Info(ctx, "topic map retrieval disabled")
		}
```
Also, why doesn't `LoadTopicsMap()` return an error? What if there was a failure to retrieve it?